<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultilevelQueueWindow</class>
 <widget class="QWidget" name="MultilevelQueueWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="pageNameLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: orange;
	font-size: 35px;
	font-family: Roboto;
	background-color: transparent;
	text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string>Multilevel queue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="pageNameLabel_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>621</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: grey;
	font-size: 35px;
	font-family: Roboto;
}</string>
   </property>
   <property name="text">
    <string> - line section (objects) work sequence</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="demoButton">
   <property name="geometry">
    <rect>
     <x>1370</x>
     <y>830</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>üëÄ Demo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>830</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>‚Üê Go back</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskListNameInput">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>620</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addRandomEventButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>380</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ûï Random event</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="multilevelTable">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>170</y>
     <width>631</width>
     <height>621</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size: 20px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
	padding-left: 30px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTaskListButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>570</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ûï  Multilevel queue</string>
   </property>
  </widget>
  <widget class="QLabel" name="multilevelNameLabel">
   <property name="geometry">
    <rect>
     <x>435</x>
     <y>110</y>
     <width>631</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #595959;
	font-size: 25px;
	font-family: Roboto;
	background-color: #d9d9d9;
	border-radius: 3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="addTaskButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>690</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>‚ûï  task</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>620</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>740</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="taskPriorInput">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>768</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>770</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>prior = </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskNameInput">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>740</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="toSelectionRadioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>165</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QLabel" name="fromStationLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>115</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
	font-size: 17px;
	background-color: #e1e1e1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="toStationLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: black;
	font-size: 17px;
	font-family: Roboto;
	background-color: #e1e1e1;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="trainLineTable">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>269</y>
     <width>261</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QHeaderView::section {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 #616161, stop: 0.5 #505050,
                                      stop: 0.6 #434343, stop:1 #656565);
    color: white;
    padding-left: 4px;
    border: 1px solid #6c6c6c;
}

QTableView {
    selection-background-color: #dedede;

	selection-color: dark-grey;

	font-size: 22px;
}


/* style the sort indicator */
QHeaderView::down-arrow {
    image: url(down_arrow.png);
}

QHeaderView::up-arrow {
    image: url(up_arrow.png);
}

QTableView::item {
	padding-left: 30px;
}</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="fromSelectionRadioButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>from</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="trainLineNameLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: #595959;
	font-size: 18px;
	font-family: Roboto;
	background-color: #d9d9d9;
	border-radius: 3px;
}</string>
   </property>
   <property name="text">
    <string>Obolonsko-Teremkivska line</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>470</y>
     <width>350</width>
     <height>4</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid black;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>110</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: #d9d9d9;
    border-radius: 3px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="worker1_taskLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 17px;
	border-bottom: 2px solid black;
	font-family: Roboto;
}</string>
    </property>
    <property name="text">
     <string>Worker 1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="worker1_takeTaskButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>‚Üì Take task</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>110</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
	background-color: #d9d9d9;
    border-radius: 3px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	font-size: 17px;
	border-bottom: 2px solid black;
	font-family: Roboto;
}</string>
    </property>
    <property name="text">
     <string>Worker 2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="worker2_taskLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="worker2_takeTaskButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1310</x>
     <y>270</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>‚Üì Take task</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>430</y>
     <width>311</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font-size: 17px;
	color: red;
	font-family: Roboto;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="demoModeLabel">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>0</y>
     <width>391</width>
     <height>91</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	color: orange;
	font-size: 35px;
	font-family: Roboto;
	background-color: transparent;
	text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
